stac_model/input.py

```diff
@@ -19,43 +20,17 @@ def validate_dimensions(self) -> Self:
 
 
 class MLMStatistic(MLMBaseModel):  # FIXME: add 'Statistics' dep from raster extension (cases required to be triggered)
-    minimum: Annotated[Optional[Number], OmitIfNone] = None
-    maximum: Annotated[Optional[Number], OmitIfNone] = None
-    mean: Annotated[Optional[Number], OmitIfNone] = None
-    stddev: Annotated[Optional[Number], OmitIfNone] = None
-    count: Annotated[Optional[int], OmitIfNone] = None
-    valid_percent: Annotated[Optional[Number], OmitIfNone] = None
+    minimum: Annotated[Number | None, OmitIfNone] = None
+    maximum: Annotated[Number | None, OmitIfNone] = None
+    mean: Annotated[Number | None, OmitIfNone] = None
+    stddev: Annotated[Number | None, OmitIfNone] = None
+    count: Annotated[int | None, OmitIfNone] = None
+    valid_percent: Annotated[Number | None, OmitIfNone] = None
 
 
-NormalizeType: TypeAlias = Optional[
-    Literal[
-        "min-max",
-        "z-score",
-        "l1",
-        "l2",
-        "l2sqr",
-        "hamming",
-        "hamming2",
-        "type-mask",
-        "relative",
-        "inf",
-    ]
-]
+NormalizeType: TypeAlias = Literal["min-max", "z-score", "l1", "l2", "l2sqr", "hamming", "hamming2", "type-mask", "relative", "inf"] | None
 
-ResizeType: TypeAlias = Optional[
-    Literal[
-        "crop",
-        "pad",
-        "interpolation-nearest",
-        "interpolation-linear",
-        "interpolation-cubic",
-        "interpolation-area",
-        "interpolation-lanczos4",
-        "interpolation-max",
-        "wrap-fill-outliers",
-        "wrap-inverse-map",
-    ]
-]
+ResizeType: TypeAlias = Literal["crop", "pad", "interpolation-nearest", "interpolation-linear", "interpolation-cubic", "interpolation-area", "interpolation-lanczos4", "interpolation-max", "wrap-fill-outliers", "wrap-inverse-map"] | None
```

Comment:
Is there an option to leave those line-by-line? They are way easier to edit that way (easier copy-paste from values edited in JSON schema).

Reply:
I added noqa to those, they are unchanged now.